.github/workflows/ci.yml

```diff
@@ -232,14 +154,15 @@ jobs:
         run: dotnet msbuild /t:UnityBuildPlayerAndroidIL2CPP /p:Configuration=Release /p:OutDir=other src/Sentry.Unity
 
       - name: Build iOS Player
-        if: matrix.os == 'macos-latest'
         run: dotnet msbuild /t:UnityBuildPlayerIOS /p:Configuration=Release /p:OutDir=other src/Sentry.Unity
 
-      - name: Build Standalone Player with IL2CPP
-        run: dotnet msbuild /t:UnityBuildStandalonePlayerIL2CPP /p:Configuration=Release /p:OutDir=other src/Sentry.Unity
+      # TODO: build for all supported desktop platforms
```

Comment:
Not sure we can build a Windows standalone player on Linux

Reply:
You were right, on Linux you can only target Windows-mono, not IL2CPP... I'm thinking of running a pre-built windows unity container... Should still be much faster then installing unity

Reply:
This ended up as a separate `windows-smoke-test` job